harper-core/src/linting/somewhat_something.rs

```diff
@@ -35,13 +34,13 @@ impl ExprLinter for SomewhatSomething {
             span,
             lint_kind: LintKind::Style,
             suggestions: vec![Suggestion::replace_with_match_case_str("something", og)],
-            message: "Use the traditional form.".to_owned(),
+            message: "Consider using `something of a` in more formal writing.".to_owned(),
             priority: 63,
         })
     }
 
     fn description(&self) -> &'static str {
-        "When describing a single instance of a noun, use `something` rather than `somewhat`."
+        "Flags the phrase `somewhat of a` in favor of `something of a`, which can be considered more traditional."
```

Comment:
```suggestion
        "Flags the phrase `somewhat of a` in favor of `something of a`, which is considered more traditional."
```
The description feels a little non-committal. I'd change it to `is considered`.